### ISSUE TITLE
Issues with rmq.log :tree/Live code reload

### ISSUE THREAD

**Issue by DiogoAndre** [infinitered/rmq#240]
Hi!

I bumped into this issue when playing with the new live code reload feature. My app has a PM::MapScreen, and when live reload is on and I try to open the screen, the app crashes:

```ruby
2015-04-16 14:19:55.765 show_me[24791:2470631] app.rb:133:in `current_view_controller:': undefined method `childViewControllers' for #<CLLocationCoordinate2D:0x11949ef10> (NoMethodError)
```

I get a similar erro if I try to ```rmq.log :tree``` on the REPL, minus the app crash.

I'm guessing they booth run the same piece of code on [app.rb:L133-L134](https://github.com/infinitered/rmq/blob/61c177be72a741e809644c00e7fb251c08be02bb/motion/ruby_motion_query/app.rb#L133-L134)

Tried a couple of simple fixes but no luck.  Any idea on how to fix, or a workaround?

Thanks!

**Comment by DiogoAndre** [comment]
Haven't been able to fix the crash yet, I what did was to add a check just to avoid crashing the app.. but it also makes the live reloading not work for the PM:MapScreen

Here's what I did:
```ruby
elsif  root_view_controller.respond_to?(:childViewControllers)  && root_view_controller.childViewControllers.count > 0
  current_view_controller(root_view_controller.childViewControllers.first)
else
  if root_view_controller.respond_to?(:childViewControllers) 
    root_view_controller
else
  nil
end
```

**Comment by twerth** [comment]
Did you mean to have 2 elses there?

**Comment by twerth** [comment]
So there is a few problems with this I've found. I'm fixing it right now. Fix will be in 1.3.4

**Comment by DiogoAndre** [comment]
Thanks!! And just to clarify.. the two elses are due to poor indenting and partial code :smiley:
